plugins/NavStars/src/NavStars.cpp

```diff
@@ -660,6 +664,26 @@ void NavStars::populateNavigationalStarsSet(void)
 				     << 102098 << 107315 << 113368;
 			break;
 		}
+		case Skywatcher:
+		{
+			// Skywatcher SynScan Hand Held Controller and SynScan Pro mobile and desktop App
+			// Sources: https://www.iceinspace.com.au/63-501-0-0-1-0.html
+			//          https://www.iceinspace.com.au/download.php?10d72300ac5d3762f0529eb11977fb68
+			starNumbers << 13847 <<  33579 <<  95947 <<  67301 <<  17702 <<  21421 << 105199
```

Comment:
My gut feeling is that `starnumbers = {13847, 33579, ...}` would be more efficient than this subsequent list buildup. It's minuscule, but maybe you change all lists accordingly while you're at it.